src/unknown.rs

```diff
@@ -25,7 +25,8 @@ unsafe impl ComInterface for IUnknown {
 
 #[repr(C)]
 pub struct abi_IUnknown {
-    pub(crate) query: extern "system" fn(RawComPtr<IUnknown>, &Guid, *mut RawPtr) -> ErrorCode,
-    pub(crate) addref: extern "system" fn(RawComPtr<IUnknown>) -> u32,
-    pub(crate) release: extern "system" fn(RawComPtr<IUnknown>) -> u32,
+    pub(crate) unknown_query:
```

Comment:
I’d also prefer for these to have their canonical names (but transformed to camel case)

Reply:
Happy to do so, I just need to prepend unknown/inspectable in order to avoid collisions as I started adding support for implementations.